gauntlet/packages/gauntlet-serum-multisig/artifacts/schemas/serum_multisig.json

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
Can you remove the schema artifact from `gauntlet-solana-contracts`? We shouldn't need it there anymore
(`gauntlet/packages/gauntlet-solana-contracts/artifacts/schemas/serum_multisig.json`)